LadyBug/Modules/Auth/Registration Module/Endpoint/RegistrationEndpoint.swift

```diff
@@ -0,0 +1,47 @@
+
+//  RegistrationEndpoint.swift
+//  LadyBug
+//
+//  Created by Mohamed Abdelhamed Ahmed on 4/17/21.
+
+
+import Moya
+
+enum RegistrationEndpoint {
+    case registration(name: String, email: String, mobile: String, password: String, passwordConfirmation: String, humanJobId: String, photo: String)
```

Comment:
We don't need to send passwordConfirmation here just check before calling endpoint that it's the same